lib/vyasa/medium/event.ex

```diff
@@ -0,0 +1,40 @@
+defmodule Vyasa.Medium.Event do
+  use Ecto.Schema
+
+  import Ecto.Changeset
+
+  alias Vyasa.Written.{Verse, Source}
+  alias Vyasa.Medium.Voice
+
+  @primary_key {:id, Ecto.UUID, autogenerate: true}
+  schema "events" do
+    field :origin, :integer
+    field :duration, :integer
+    field :phase, :string
```

Comment:
throwback to the comment about phases being used to encode things like instrumentals and such